### PR TITLE
Force create /usr/lib/systemd/system

### PR DESCRIPTION
**Pull request by tdemaret**
Add this because minikube vm-driver none doesn't work on Debian and Ubuntu, because this directory does not exist.

### CODE REVIEW

**Comment by aaron-prindle** [comment]
@minikube-bot ok to test

**Comment by r2d4** [comment]
I don't think this will actually fix the issue.  Minikube actually requires a full systemd install

**Comment by aaron-prindle** [comment]
This is a valid issue, I encountered it running minikube none on Ubuntu.  We expect a directory this directory to exists even though it doesn't on some systemd installations.  Everything works properly if this directory is added.

**Comment by tdemaret** [comment]
Yes, this directory is not mandatory with all systemd installation. With Debian minimal and Ubuntu minimal for example, it doesn't exist.

File: pkg/minikube/cluster/commands.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func GetStartCommandSystemd(kubernetesConfig KubernetesConfig, localkubeStartCmd
 	if err := t.Execute(&buf, data); err != nil {
 		return "", err
 	}
-    Mkdir("/usr/lib/systemd/system/",FileMode(0755))
+    os.Mkdir("/usr/lib/systemd/system/",os.FileMode(0755))
```
Comment (aaron-prindle) [inline pkg/minikube/cluster/commands.go]: Can you move this folder creation to https://github.com/kubernetes/minikube/blob/master/pkg/minikube/machine/drivers/none/none.go#L158
It makes more sense to be part of the none driver start function.

**Comment by tdemaret** [comment]
Ok I have change in none driver.

Thanks

**Comment by aaron-prindle** [comment]
Can you you rebase your commit and push it once more?  There was a merge error at HEAD that has been resolved.  This appears to be why your tests are failing here.  Thanks.

**Comment by dlorenc** [comment]
Hey,

It looks like gofmt is still failing on this. Could you run:

gofmt -w -s pkg/minikube/cluster/commands.go
gofmt -w -s pkg/minikube/machine/drivers/none.go

File: pkg/minikube/machine/drivers/none/none.go [hunk 2]
```diff
@@ -153,6 +153,7 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 }
 
 func (d *Driver) Start() error {
+	os.Mkdir("/usr/lib/systemd/system/",os.FileMode(0755))
```
Comment (dlorenc) [inline pkg/minikube/machine/drivers/none/none.go]: Would you mind switching this to use MkdirAll and add an error check?

This should work:
```go
	if err := os.MkdirAll("/usr/lib/systemd/system/", os.FileMode(0755)); err != nil {
		return err
	}
```

**Comment by aaron-prindle** [comment]
It appears that the travis tests are failing a gofmt check that we have:
```
Checking gofmt...
Gofmt errors in files: pkg/minikube/cluster/commands.go
pkg/minikube/machine/drivers/none/none.go
```
Running the following commands:
```
gofmt -w -s pkg/minikube/cluster/commands.go
gofmt -w -s pkg/minikube/machine/drivers/none.go

```
should resolve this.

**Comment by aaron-prindle** [comment]
LGTM.  The two failing tests appear to be flakes.  Thanks for the PR!

**Pull request merged**